packages/cli/test/functional/test_site/sub_site/nested_sub_site/testNunjucksPathResolvingInclude.md

```diff
@@ -1,3 +1,3 @@
-This is content from testNunjucksPathResolvingInclude.mbdf located at `sub_site/nested_sub_site`
+This is content from testNunjucksPathResolvingInclude.md located at `sub_site/nested_sub_site`
```

Comment:
```suggestion
This is content from `testNunjucksPathResolvingInclude.md` located at `sub_site/nested_sub_site`
```

Might want to add a code fence here to prevent it from converting to a link in the expected test files :)